### ISSUE TITLE
Make Work for RSocrata

### ISSUE THREAD

**Issue by stuagano** [codeforatlanta/apd-crime-data#1]
`Get and clean Atlanta Police Department Data
get.apd.data <- function(selector.text, save.location){
     require(rvest)
     require(dplyr)
     temp <- tempfile()
     download.file(
          read_html("http://www.atlantapd.org/crimedatadownloads.aspx") %>% 
               html_nodes(selector.text) %>% 
               html_attr("href") %>% 
               paste0("http://www.atlantapd.org/", .),
          destfile = temp)
     # Save filename of csv crime data raw file, for later use
     crime.data.filename <- unzip(temp, list = TRUE)$Name
     # Unzip crime data to "data" folder and close temp file placeholder
     unzip(temp, exdir = save.location)
     unlink(temp)
     crime.data.filename`

Was there a reason why you did it this way instead of using SODA? Other PD's could definitely take advantage of this if it was using RSocrata from Chicago.

**Comment by bbrewington** [comment]
Good point.  That will probably be the best route; I think we need to clean up the data in Socrata, though.  Looks like there's a difference starting in 2015:

### Socrata Data:

<img width="936" alt="socrata crimes by rpt_date" src="https://cloud.githubusercontent.com/assets/10573749/15491150/cef12e2c-213b-11e6-9272-05770b6f20f0.PNG">

### COBRA051916.csv file:

![num crimes by rpt_date - year month](https://cloud.githubusercontent.com/assets/10573749/15490751/e67a2236-2138-11e6-9989-a0a7842f7e85.png)

### Possible Cause: import tool is duplicating rows

<img width="423" alt="socrata apd crime - rows affected repeating" src="https://cloud.githubusercontent.com/assets/10573749/15491176/ee426fde-213b-11e6-80f8-35544ec3f4a9.PNG">

**Comment by bbrewington** [comment]
@stuagano would it make more sense to do the ETL via https://dev.socrata.com/connectors/pentaho-kettle.html instead of the R script in this repo?  Maybe if there was a way to schedule an R ETL job it could work.  What do you think?

Here's the ETL current process (it would be nice to automate the last step of publishing cleaned data to Socrata):
<img width="510" alt="etl process" src="https://cloud.githubusercontent.com/assets/10573749/15491400/0f9bf2b6-213e-11e6-8fc2-2be2a1be7828.PNG">

edit: note to self - https://github.com/jwijffels/taskscheduleR
